lib/cryptoexchange/exchanges/bybit/services/market.rb

```diff
@@ -27,7 +27,7 @@ def adapt_all(output)
             market_pair = Cryptoexchange::Models::MarketPair.new(
               base: base,
               target: target,
-              market: Btcnext::Market::NAME
+              market: Bybit::Market::NAME
```

Comment:
is this an unrelated mistake ?

Reply:
yes noticed this mistake from past implementation